blackbox/docs/release_notes/1.1.3.txt

```diff
@@ -0,0 +1,73 @@
+.. _version_1.1.3:
+
+=============
+Version 1.1.3
+=============
+
+Released on 2017/05/09.
+
+.. NOTE::
+
+   If you are upgrading a cluster, you must be running CrateDB
+   :ref:`version_1.0.2` or higher before you upgrade to 1.1.3.
+
+   If you want to perform a :ref:`cluster_upgrade`, your current CrateDB
+   version number must be :ref:`version_1.1.2`. Any upgrade from a version
```

Comment:
i agree. not sure if we really need this. imho an upgrade should work from everything in 1.0.x. you need to clarify this.